WcaOnRails/db/migrate/20220529123854_email_default_change.rb

```diff
@@ -0,0 +1,7 @@
+class EmailDefaultChange < ActiveRecord::Migration[6.1]
```

Comment:
Please give a more concise name to the migration. Which email default is being changed?

Reply:
Do you mean a more descriptive name? Making it more concise would make it hard to specify which default is being changed.